lib/streamlit/hashing.py

```diff
@@ -31,6 +31,8 @@
 import tempfile
 import threading
 
+import numpy
```

Comment:
as `np`?

Reply:
We refer to `numpy` as `numpy` in some comments and string in here, so this seems cleaner to me... No strong opinion either.